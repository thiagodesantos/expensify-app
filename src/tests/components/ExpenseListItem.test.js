import React from 'react';
import { shallow } from 'enzyme';
import expenses from '../fixtures/expenses'
import ExpenseListItem from '../../components/ExpenseListItem';

test('Deveria renderizar o ExpenseListItem corretamente',() =>{
    const wrapper = shallow(<ExpenseListItem  {...expenses[0]}/>);
    expect(wrapper).toMatchSnapshot();
});