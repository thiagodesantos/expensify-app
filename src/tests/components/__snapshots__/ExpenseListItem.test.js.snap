// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deveria renderizar o ExpenseListItem corretamente 1`] = `
<div>
  <Link
    to="/editar/1"
  >
    <h3>
      Balas
    </h3>
  </Link>
  <p>
    $1.95
    -
    31st December, 1969
  </p>
</div>
`;
